LibreNMS/Modules/Ospf.php

```diff
@@ -87,6 +87,11 @@ public function poll(OS $os, DataStorageInterface $datastore): void
                 if (empty($ospf_entry['ospfRouterId'])) {
                     continue; // skip invalid data
                 }
+                foreach (['ospfRxNewLsas', 'ospfOriginateNewLsas', 'ospfAreaBdrRtrStatus', 'ospfExternLsaCount', 'ospfTOSSupport', 'ospfExternLsaCksumSum', 'ospfExternLsaCount', 'ospfASBdrRtrStatus', 'ospfVersionNumber', 'ospfAdminStat'] as $column) {
```

Comment:
ospfExternLsaCount is twice on the list

Reply:
On my way to fix that. I suppose double check is not necessary :D Did you check it in your instance to confirm the issue is gone ?